luma/core/lib.py

```diff
@@ -20,8 +20,9 @@ def __rpi_gpio__(self):
     # a Raspberry Pi... this is imported here so we can swap out the
     # implementation for a mock
     try:
-        import RPi.GPIO
-        return RPi.GPIO
+        import RPi.GPIO as GPIO
+        GPIO.setmode(GPIO.BCM)
```

Comment:
Somewhat related to the ticket you just closed (#2): If you _dont_ supply a gpio= override, then the behaviour is unchanged from present - it sets up the RPi.GPIO with BCM mode.

For those times when gpio= is provided, the chances are the reason is that it needs to integrate with an existing codebase that uses a RPi.GPIO which has already been setup and initialized: we shouldn't assume BCM is the right mode for all applications & should let the caller choose.

Plus ... I have an alternative [OPi.GPIO](https://github.com/rm-hull/OPi.GPIO) implementation under development where it make more sense to use the SUNXI pin mappings with the Orange Pi Zero... [the implementation does support translating BCM to native pin mappings though]

Same goes for the CHIP_IO, which I suspect _probably_ just ignores the `GPIO.setmode(GPIO.BCM)` (haven't checked)